### ISSUE TITLE
[Shop Feature] Show lowest price for products with variables

### ISSUE THREAD

**Issue by chalkandmoss** [facebookincubator/facebook-for-woocommerce#233]
WooCommerce displays the lowest price for variable products on the front page of a product (e.g. where small, medium and large are available for vases). But on the Facebook shop, it seems to choose the price at random, which will confuse shoppers. For some products it displays a lower priced selection, sometimes the higher. 
My site is www.chalkandmoss.com

**Comment by sammul** [comment]
Thanks @chalkandmoss , I found an example on your shop: https://www.prod.facebook.com/commerce/products/1967716909924818/

Looks like a feature request on Facebook shop side. We will forward this to the shop team.

**Comment by mengyingdu** [comment]
Hi @chalkandmoss ,

There is workaround for you. On facebook, the default price is the last variant when you create this variable product. If you want to show the price with the lowest price, you can delete the one first and then create it again (so it will be shown as the default one on facebook side)

Thanks.

**Comment by chalkandmoss** [comment]
Thank you for the tip. As this would require a lot of re-work on all the products, I think I prefer to wait for your team to amend the plugin to mirror the woocommerce price. Do you have an idea when this might be? Thanks a million for your support and consideration!

**Comment by chalkandmoss** [comment]
Apologies, I've just noticed that Woocommerce actually shows the price RANGE, not the lowest price as I previously thought. So it might say £50-120 for one product, if there's a small, medium and large. It would be great if the Facebook shop page could mirror this. Here's a screenshot from my Woocommerce store at chalkandmoss.com.
![screen shot 2018-01-30 at 20 54 52](https://user-images.githubusercontent.com/35746921/35590818-e050d924-05ff-11e8-8794-d5215cc9e135.png)

**Comment by mengyingdu** [comment]
Thanks for follow-up with detailed explanation. I will flag this to our shop team.

**Comment by mengyingdu** [comment]
Hi @chalkandmoss,

We updated the default product feature. If you set the one (lowest) variation as default in woocommerce product editor, then click on 'update'. It will automatically be set as the default one on your facebook shop. 

Given we are swapped, i don't think the new logic for default display now require too much manual efforts. Upload screen shot to clarify:
![screen shot 2018-06-26 at 5 30 08 pm](https://user-images.githubusercontent.com/24535640/41946306-a3e0b998-7966-11e8-8be7-19441a932ab7.png)



Thanks.

**Comment by chalkandmoss** [comment]
Thanks a lot for this update. This solves the issue of Facebook showing the most expensive price (where Woocommerce shows the cheapest price by default). However, it leaves the issue that Facebook doesn't show the "product image" from woocommerce, instead showing one of the variation images from the gallery of images, in this case now the one marked default. I am very focused on lifestyle images in my feed, so the product image is almost always a lifestyle image, as are my social post images. So it's still not possible for Facebook to show the product image, instead of a variation image from the gallery? Many thanks!

**Comment by AspireFox** [comment]
@chalkandmoss I had this EXACT same issue. After wasting a ton of time trying to fix it, and researching I came across your post. Your post helped me correct the random variation showing by setting the default form value for the lowest cost variation! Thanks!!  I also solved the random picture issue by  choosing "use parent image" located under the Facebook tab under product data then updating. Took awhile as there was no way to do this in bulk but worked! Hopefully this helps if you have not found a solution as yet. Cheers!
